src/coreclr/scripts/superpmi.py

```diff
@@ -903,21 +902,6 @@ def __init__(self, coreclr_args):
 
         """
 
-        if coreclr_args.host_os == "OSX":
```

Comment:
Why did you remove these? the collection_shim_name and corerun_tool_name are still used.

Reply:
Sorry, that was by mistake. I meant to delete just a single line related  to `crossgen`.